ports/assaultshell/port.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 3,
+  "name": "assaultshell.zip",
+  "items": [
+    "ASSAULT SHELL.sh",
+    "assaultshell/"
```

Comment:
```suggestion
    "assaultshell"
```

Reply:
The / is literally in the port.json example.

Reply:
I totally don't mind taking it out if we all agree it's out but the example must be changed🙂